src/clipper/web-extension/normalize.css

```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
I thought nowadays the differences between browsers is negible. Does it look broken?

Reply:
yeah, especially for buttons and input elements. Could also normalize those manually but this was easier and more future proof